.github/workflows/start-vulnerability-updater.yaml

```diff
@@ -0,0 +1,61 @@
+name: Vulnerability Updater
+on:
+  push:
+    branches:
+      - "yli3/**"
+  schedule:
+    - cron: "0 */3 * * *"
+
+jobs:
+  read-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.output-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Read versions
+        id: versions
+        run: |
+          VERSIONS=$(cat scanner/updater/version/RELEASE_VERSION)
+          echo "$VERSIONS" | grep -Eo '([0-9]+\.[0-9]+\.[0-9]+|[0-9]+\.[0-9]+\.x-[0-9]+-g[a-f0-9]+)' > versions.txt
+
+      - name: Generate matrix JSON
+        id: output-matrix
+        run: |
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+          MATRIX=$(jq -Rs '{ versions: split("\n")[:-1] }' versions.txt)
+          echo "matrix<<EOF"$'\n'"${MATRIX}"$'\n'EOF >> "${GITHUB_OUTPUT}"
+          echo "${MATRIX}"
```

Comment:
You are basing this on [the docs](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-of-a-multiline-string)?

I think this would be simpler to read and maintain. And you can get rid of the previous step, too.

Let me know:

```suggestion
          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
          echo "matrix<<$EOF" >> $GITHUB_OUTPUT
          sed -n '/^\s*\(#.*\)\?$/!p' scanner/updater/version/RELEASE_VERSION | jq -Rs '{ versions: split("\n")[:-1] } | tee -a $GITHUB_OUTPUT
          echo $EOF >> $GITHUB_OUTPUT
```